auslib/db.py

```diff
@@ -2634,7 +2664,8 @@ def setupChangeMonitors(self, relayhost, port, username, password, to_addr, from
                 self.productRequiredSignoffs, self.productRequiredSignoffs.scheduled_changes, self.productRequiredSignoffs.scheduled_changes.signoffs,
                 self.permissionsRequiredSignoffs, self.permissionsRequiredSignoffs.scheduled_changes,
                 self.permissionsRequiredSignoffs.scheduled_changes.signoffs,
-                self.releases.scheduled_changes, self.releases.scheduled_changes.signoffs,
+                self.releases, self.releases.scheduled_changes, self.releases.scheduled_changes.signoffs,
```

Comment:
What's the idea with adding self.releases here ? We previously disabled notifications for those because there are ~500 changes each time we do a new release, so more noise than signal. Definitely want read only state changes though.